Code/EditorPlugins/Scene/EnginePluginScene/RenderPipeline/EditorSelectedObjectsExtractor.cpp

```diff
@@ -84,6 +84,19 @@ void ezEditorSelectedObjectsExtractor::Extract(
   ezSelectedObjectsExtractorBase::Extract(view, visibleObjects, ref_extractedRenderData);
 }
 
+ezResult ezEditorSelectedObjectsExtractor::Serialize(ezStreamWriter& inout_stream) const
+{
+  EZ_SUCCEED_OR_RETURN(SUPER::Serialize(inout_stream));
+  return EZ_SUCCESS;
+}
+
+ezResult ezEditorSelectedObjectsExtractor::Deserialize(ezStreamReader& inout_stream)
+{
+  EZ_SUCCEED_OR_RETURN(SUPER::Deserialize(inout_stream));
+  // const ezUInt32 uiVersion = ezTypeVersionReadContext::GetContext()->GetTypeVersion(GetStaticRTTI());
```

Comment:
instead of commenting this out it would be better to use `EZ_IGNORE_UNUSED`

Reply:
That would require including `#include <Foundation/IO/TypeVersionContext.h>` in every of those 28 files, it that really better?

Reply:
It's better because then the code actually gets compiled, and it's easier to find errors in case anything with this version fetching ever changes.

Reply:
Replaced all instances.